libcloud/compute/drivers/ecs.py

```diff
@@ -768,6 +768,27 @@ def ex_stop_node(self, node, ex_force_stop=False):
         return resp.success() and \
             self._wait_until_state([node], NodeState.STOPPED)
 
+    def ex_create_security_groups(self, ex_description=None,
```

Comment:
For consistency with other drivers, please name it `ex_create_security_group` (remove `s` suffix) and remove `ex_` prefix from argument names since method name is already prefixed.

This way signature should similar to the AWS one -> `def ex_create_security_group(self, description=None, client_token=None)`